release-notes.md

```diff
@@ -0,0 +1,55 @@
+# Release v0.7.1
+
+## New features
+* Add [alert manager](prometheus/doc/alert-manager.md), defined [several rules](https://github.com/Microsoft/pai/tree/pai-0.7.y/prometheus/prometheus-alert) for alert. Operator will receive email notification if cluster has some problem.
+
+## Improvements
+* [Optimize](https://github.com/Microsoft/pai/pull/1021) boot speed of web portal.
+* Do [not delete](https://github.com/Microsoft/pai/pull/1038) etcd data by default when upgrading kubernetes.
```

Comment:
**I tuned the message a little bit:**

Optimized the boot speed of web portal - [PR 1021](https://github.com/Microsoft/pai/pull/1021);
Improved the Kubernetes upgrade experience, PAI admin is no more required to delete ETCD data when upgrading Kubernetes - [PR 1038](https://github.com/Microsoft/pai/pull/1038)